go/private/tools/path.bzl

```diff
@@ -67,9 +67,19 @@ def _go_path_impl(ctx):
   manifest_entries = []
   manifest_entry_map = {}
   for pkg in pkg_map.values():
-    for f in pkg.srcs + pkg.data:
-      _add_manifest_entry(manifest_entries, manifest_entry_map, inputs,
-                          f, pkg.dir + "/" + f.basename)
+    for f in pkg.srcs:
+      dst = pkg.dir + "/" + f.basename
+      _add_manifest_entry(manifest_entries, manifest_entry_map, inputs, f, dst)
+  if ctx.attr.include_data:
+    for pkg in pkg_map.values():
+      for f in pkg.data:
+        parts = f.path.split("/")
```

Comment:
wouldn't it be easier to just split on /testdata/

Reply:
In this case yes. As a habit though, when I'm looking for a specific path component, I prefer to split the path and look at the parts. It avoids corner cases with the first and last components.